packages/fx-core/src/common/samples-config.json

```diff
@@ -1,6 +1,6 @@
 {
-    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/main/",
-    "defaultPackageLink": "https://github.com/OfficeDev/TeamsFx-Samples/archive/refs/heads/main.zip",
+    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/v2/",
```

Comment:
Is this a stable branch?

Reply:
Yes, it will replace with the main branch for the new releases of Teams Toolkit.